### DISCUSSION TITLE
Help with reported "unsafe" files

### DISCUSSION THREAD

**Discussion by Ghodeus** [Bing-su/adetailer#476] [General]
Hi there,

I notice that 5 files "person_yolov8n-seg.pt , deepfashion2_yolov8s-seg.pt , person_yolov8s-seg.pt , person_yolov8m-seg.pt , clothing_poor_yolov8s-seg.pt" have been flagged and reported unsafe. You can see that upon visiting the main repository. Could you share info on what this flag could be? [Control+Alt+AI](https://www.youtube.com/@controlaltai) did a really cool video of Adetailer and it's great. Thanks for the work.

**Comment by Bing-su** [comment]
https://huggingface.co/docs/hub/security-pickle

The file uploaded to huggingface was created by me.
